packages/graphile-build-pg/src/QueryBuilder.js

```diff
@@ -874,6 +907,32 @@ order by (row_number() over (partition by 1)) desc`;
     this.lock("selectCursor");
     this.lock("select");
   }
+  buildChild() {
+    const options = { supportsJSONB: this.supportsJSONB };
+    const child = new QueryBuilder(options, this.context, this.rootValue);
+    child.parentQueryBuilder = this;
+    return child;
+  }
+  buildNamedChildFrom(
+    name: string,
```

Comment:
Let's allow `name` to be `string | symbol` because symbols are guaranteed to be unique. This could end up being pretty handy... Maybe.